source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -0,0 +1,111 @@
+.. _telemetry-enable-disable:
+
+Enable and Disable Telemetry in Clear Linux

```

Comment:
Please use sentence case for headings: Enable or disable telemetry in Clear Linux

Reply:
done